.gitignore

```diff
@@ -3,6 +3,12 @@ BionetSolver/0.0.6/include/sbmlsolver/config.h
 CompuCell3D/core/BasicUtils/BasicPluginManager.h
 CompuCell3D/core/pythonSetupScripts/Version.py
 RoadRunner/ThirdParty/libsbml/src/sbml/common/libsbml-config-common.h
+CompuCell3D/core/MaBoSS/engine/src/BooleanGrammar.cc
```

Comment:
Do we need this? I thought that BionetSolver was not a part of CC3D anymore . But maybe you are repurposing it for something else

Reply:
The MaBoSS-related entries where for generated scripts that are site-specific. I removed those and outdated entries for RoadRunner and BionetSolver in https://github.com/CompuCell3D/CompuCell3D/pull/422/commits/9fa97875db69d524585f4b0df9a2da543683d3cf and https://github.com/CompuCell3D/CompuCell3D/pull/422/commits/974cd0d5c08ca357c87e9926b4d300a843bc6b0d.